net.ltgt.gwt.maven

yurloc opened the pull request.

yurloc committed:
PLANNER-1026 [ui] No sources missing

yurloc committed:
PLANNER-1026 [app] Remove client stuff (pom.xml)

yurloc committed:
PLANNER-1026 [ui] Write ServerMarshallerFactory* to WEB-INF/classes

ge0ffrey commented on optaplanner-wb-webapp-war/.gitignore:
```diff
@@ -0,0 +1,16 @@
+/local
+/repositories
+/src/main/gwt-unitCache/
+/src/main/webapp/WEB-INF/classes/
```
war generated files should go into target/....war/WEB-INF/classes ?

yurloc replied:
Good question. Currently the ignore is justified because build output directory is set to `src/main/webapp/WEB-INF/classes`. Don't know if that's necessary. I'll see if we can get rid of that later.

yurloc commented on optaplanner-wb-webapp-war/pom.xml:
```diff
@@ -0,0 +1,833 @@
+
+  <build>
+    <finalName>${project.artifactId}</finalName>
+    <outputDirectory>src/main/webapp/WEB-INF/classes</outputDirectory>
```
@ge0ffrey Here is why the gitignore entry is needed. This is a copied from the former optaplanner-wb-webapp module. Let's get rid of this later.

yurloc committed:
PLANNER-1026 [ui] Source maps & pretty style

yurloc committed:
PLANNER-1026 [ui] Copy original optaplanner-wb-webapp

yurloc committed:
PLANNER-1026 [ui] Remove server stuff

kiereleaseuser committed:
upgraded to next development version

yurloc committed:
PLANNER-1026 [app] Remove client stuff (src)

kiereleaseuser committed:
upgraded src/main/config/zanata.xml to link to new project 7.8.0 on https://vendors.zanata.redhat.com/ (#267)

adrielparedes committed:
[AF-1087]: Added VisibleAsset Qualifier to ResourceTypeDefinitions (#269)

yurloc committed:
PLANNER-1026 [ui+app] Final touches

yurloc committed:
PLANNER-1026 [ui] API sources for model marshallers

mbarkley committed:
AF-1084: Perform simple indexing of filenames separate from full indexing that requires a build (#265)

yurloc committed:
PLANNER-1026 [SDM] Move templates to src/main/resources

This is required when running GWT codeserver from command line as an
aggregator goal. In this setting, templates need to be present in the
project modules in target/classes. They were missing because compiler
plugin only processes .java files. Other resources are moved by
resources plugin from src/main/resources.

yurloc committed:
PLANNER-1026 [SDM] Fix scopes to work with SDM

yurloc committed:
PLANNER-1026 [SDM] Fix gwt-maven-plugin configuration to allow SDM

1. Start CodeServer from project root

    mvn gwt:codeserver -pl optaplanner-wb-ui -am

2. In a new terminal window, start the backend

    cd optaplanner-wb-webapp
    mvn wildfly:start wildfly:deploy-only

When shutting down, send ^C to the CodeServer, run mvn wildfly:shutdown
to stop wildfly.
